src/interface/src/app/map/map-control-panel/map-control-panel.component.html

```diff
@@ -152,6 +153,7 @@ <h3><b>Map Views</b></h3>
             </mat-radio-group>
           </div>
         </mat-expansion-panel>
+      </ng-container>
```

Comment:
nit on spacing; appears unaligned w/ line 138.

Reply:
Wow, this just got bumped up to 2 story points :( (jk)